public/index.html

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
It would be great to have a screenshot or something around what this looks . I would suggest a screen recording loaded into youtube as you have ALOT of functionality with the logging in etc